SU2_CFD/include/numerics/CNumerics.hpp

```diff
@@ -698,6 +703,28 @@ class CNumerics {
    */
   virtual void SetCrossDiff(su2double val_CDkw_i) {/* empty */};
 
+  /*!
+  * \brief Get the final separation intermittency for the LM model.
+  */
+  inline su2double GetIntermittencySep() const { return intermittency_sep_i; }
+
+  /*!
+   * \brief Set the value of the effective intermittency for the LM model.
+   * \param[in] intermittency_eff_i - Value of the effective intermittency at point i.
+   * \param[in] intermittency_eff_j - Value of the effective intermittency at point j.
+   */
+  virtual void SetIntermittencyEff(su2double val_intermittency_eff_i, su2double val_intermittency_eff_j) {
```

Comment:
If the member variables are in CNumerics, these Set functions should not need to be virtual.
```suggestion
  void SetIntermittencyEff(su2double val_intermittency_eff_i, su2double val_intermittency_eff_j) {
```

Reply:
This applies to the other functions you added to CNumerics